### DISCUSSION TITLE
Attacks for Decision Tree Regressor

### DISCUSSION THREAD

**Discussion by pt3th** [Trusted-AI/adversarial-robustness-toolbox#1744] [Q&A] [answered]
Hi, thanks for the great work. I was looking for the attack that can be used for ScikitlearnDecisionTreeRegressor, but I failed to do so. Is such an attack available within this package?

**Comment by beat-buesser** [comment, marked as answer]
Hi @pt3th We don't have yet a specific attack on `ScikitlearnDecisionTreeRegressor`, but it currently used in inference attacks.
